new/release-cadence.md

```diff
@@ -8,14 +8,34 @@ Type: Informational
 
 # Chef Release Cadence
 
-Chef is released once per month, in the first week of the month unless delayed
-by the Chef Software Release Engineering team.
+Chef follows [Semantic Versioning](https://semver.org/) for releases. Major
+versions (eg. 11.x -> 12.x) will include backwards-incompatible changes, minor
+versions (eg 12.1 -> 12.2) will include new features and bug fixes but will be
+backwards-compatible to the best of our ability. Patch versions are governed
+by [RFC 47](rfc047-release-process.md).
+
+Chef feature releases are promoted to the stable channel once per month, in the
+second week of the month unless delayed by the Chef Software Release Engineering
```

Comment:
Also happy for a better way to say "Nth week unless shit happens" that sounds reasonable, don't want to throw releng under the bus, something that balances standard expectations with reality happening would be good.